firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/core/CrashlyticsController.java

```diff
@@ -858,6 +858,9 @@ private void doCloseSessions(int maxCustomExceptionEvents, boolean includeCurren
 
     if (includeCurrent) {
       reportingCoordinator.onEndSession();
+    } else if (nativeComponent.hasCrashDataForSession(mostRecentSessionIdToClose)) {
+      finalizePreviousNativeSession(mostRecentSessionIdToClose);
+      nativeComponent.finalizeSession(mostRecentSessionIdToClose);
```

Comment:
before we were returning a boolean and logging based on it, do we need to do an equivalent?

Reply:
Yeah, no harm in that.